src/initurb.F90

```diff
@@ -135,6 +135,8 @@ subroutine initurb
    enddo
   enddo
  enddo
+ deallocate(ranf_av)
+
```

Comment:
I dont recall why I put this here, probably some constraint such that I couldnt put it in the main deallocate routine above